app/views/account/pages/_save.js.erb

```diff
@@ -0,0 +1,9 @@
+$("ul.errors").html("")
+<% if @page.errors.any? %>
+  <% @page.errors.full_messages.each do |message| %>
+    $("ul.errors").append($("<li />").html("<%= message.html_safe %>"))
```

Comment:
this is wrong.

re-render form, do not show errors like this. simple_form already showing errors.

Reply:
the problem is i do not understand how to realize flash works after the page created, it only shows on the next refresh